tests/utils/pvc.go

```diff
@@ -136,8 +137,8 @@ func NewPVCDefinitionWithSelector(pvcName, size string, selector map[string]stri
 // AnnSecret
 // AnnCloneRequest
 // You can also pass in any label you want.
-func NewPVCDefinition(pvcName string, size string, annotations, labels map[string]string) *k8sv1.PersistentVolumeClaim {
-	return &k8sv1.PersistentVolumeClaim{
+func NewPVCDefinition(pvcName string, size string, annotations, labels map[string]string /*volumeMode corev1.PersistentVolumeMode*/) *k8sv1.PersistentVolumeClaim {
```

Comment:
Why is there commented out parameter in the function definition?

Reply:
left over. removed it.